app/src/components/Menu.tsx

```diff
@@ -1,17 +1,17 @@
 import {
   IonContent,
+  IonHeader,
   IonIcon,
+  IonImg,
   IonItem,
   IonLabel,
   IonList,
-  IonListHeader,
   IonMenu,
   IonMenuToggle,
-  IonNote,
 } from '@ionic/react';
 
 import { useLocation } from 'react-router-dom';
-import { archiveOutline, archiveSharp, bookmarkOutline, heartOutline, heartSharp, mailOutline, mailSharp, paperPlaneOutline, paperPlaneSharp, trashOutline, trashSharp, warningOutline, warningSharp } from 'ionicons/icons';
+import { cubeOutline, cubeSharp } from 'ionicons/icons';
```

Comment:
Above you have used both ionic icons and font awesome? I dunno how different they are but should we be sticking to one or other?

Reply:
Same applies to all other files but I cannot be bothered to retype this

Reply:
This one was missed sorry! I moved over to FA Icons as Ionic icons are missing quite a few basic ones that we want

Reply:
No worries - no need to apologise lol